ext/standard/basic_functions.stub.php

```diff
@@ -823,13 +823,13 @@ function ftell($stream): int|false {}
 function fflush($stream): bool {}
 
 /** @param resource $stream */
-function fwrite($stream, string $string, ?int $max_length = null): int|false {}
+function fwrite($stream, string $data, ?int $max_length = null): int|false {}
```

Comment:
I'm not sure I see a compelling reason to deviate from the usual `$string` name here. For that matter, if we s/$data/$string in all those gz APIs, I don't think things get worse. Do they?

Reply:
I'm not opposed to use `$string`, but I'd prefer `$data` much more than `$string` since it describes the purpose of the parameter much better. And actually, we used `$data` in case of `SessionHandler::write()` and `shmop_write()`, that's why I'd like to follow the practice here as well

Reply:
Okay, I'm fine with that. It might make sense to also adjust file_put_contents() while you're already here.

Reply:
I've just added this